src/test/kotlin/com/exactpro/th2/check1/rule/sequence/TestSequenceCheckTask.kt

```diff
@@ -112,9 +112,17 @@ class TestSequenceCheckTask : AbstractCheckTaskTest() {
 
         sequenceCheckRuleTask(parentEventID, messageStream, checkOrder).begin()
 
-        val batchRequest = awaitEventBatchRequest(1000L, 5)
+        val batchRequest = awaitEventBatchRequest(1000L, 6)
```

Comment:
Why did we change the number of invocations?

Reply:
The previous version has a bug related to union events with children and without children.
Please have a look schema
```
        /*
            checkSequenceRule
              preFiltering
                Verification x 3
              checkMessages
                Verification x 3
              checkSequence
         */
```
The previous version unions checkMessages and checkSequence events